client/state/current-user/schema.js

```diff
@@ -0,0 +1,4 @@
+export default {
+	type: [ 'integer', 'null' ],
+	minimum: 0
+}
```

Comment:
Arguing the smallest of points, but I wonder why ESLint doesn't identify this as needing a semi-colon? In my mind it should I'd think.